ujjwal/src/Pages/LoginPage/ForgotPassword.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+
+function ForgotPassword({ onBackToLogin, handleSubmit, error }) {
+    return (
+        <div>
+            <div className='flex items-center justify-center'>
+                <div className='rounded-lg max-w-md w-full'>
+                    <div className='p-6'>
+                        <h1 className='text-xl font-semibold text-center'>Forgot Your Password?</h1>
+                    </div>
+                    <form onSubmit={handleSubmit} className='pl-6 pr-6 mb-36'>
+                        <p className='text-sm text-center'>Enter email address associated with your account and you will receive an email to reset your password.</p>
```

Comment:
Here almost all the child elements of the div are requiring text-sm so you can provide it in parent itself instead of children.